.github/workflows/build_and_run_tests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     name: Build and Tests
     strategy:
       matrix:
-        os: [ ubuntu-20.04, ubuntu-22.04 ]
+        os: [ ubuntu-20.04, ubuntu-22.04, ubuntu-24.04 ]
```

Comment:
I would drop 22.04, like we just did with `taskolib`.

Reply:
I'll leave it in for now. It adds one more compiler version to the mix and doesn't cost anything. I would rather get rid of Ubuntu 20 as fast as possible.